handbook/product/README.md

```diff
@@ -122,17 +122,15 @@ Bugs are always prioritized. (Fleet takes quality and stability [very seriously]
 
 If a bug is unreleased or [critical](https://fleetdm.com/handbook/engineering#critical-bugs), it is addressed in the current sprint. Otherwise, it may be prioritized and estimated for the next sprint. If a bug [requires drafting](https://fleetdm.com/handbook/engineering#in-product-drafting-as-needed) to determine the expected functionality, the bug should undergo [expedited drafting](#expedited-drafting). 
 
-If a bug is not addressed within six weeks, it is [sent to the product team for triage](https://fleetdm.com/handbook/engineering#in-engineering). Each sprint, the Head of Product meets with the group product managers to review these bugs (churned bug review). In this session, bugs are categorized as follows:
-- **Schedule**: the bug should be prioritized in the next sprint.
-- **Needs prioritization**: the bug will likely not be worked on within the next six weeks, but it is still a valid bug.
+If a bug is not addressed within six weeks, it is [sent to the product team for triage](https://fleetdm.com/handbook/engineering#in-engineering). Each sprint, the Head of Product design reviews these bugs. Bugs are categorized as follows:
```

Comment:
```suggestion
If a bug is not addressed within six weeks, it is [sent to the product team for triage](https://fleetdm.com/handbook/engineering#in-engineering). Each sprint, the Director of Product Development reviews these bugs. Bugs are categorized as follows:
```

@lukeheath after reading through the age out process, maybe this should be you? 

This way, the entire process is owned and driven by one person (less handoffs).

Reply:
cc @mikermcneil

Reply:
UPDATE: I spoke to Mike. We decided that Product (Noah) will own reviewing churned bugs to prepare for "Churned bug review" call. Noah and @lukeheath will meet at "Churned bug review" to align on which bugs we will continue to work on.

(noahtalerman)